webrender_traits/src/api.rs

```diff
@@ -248,6 +248,10 @@ impl RenderApi {
         self.api_sender.send(msg).unwrap();
     }
 
+    pub fn shut_down(&self) {
```

Comment:
Yeah, I'm not found of being able to shutdown before all of the api senders are gone. Worth doing as a followup (this blocks some gecko work so I'd prefer to iterate towards a good solution rather than take the time to figure out the perfect one the first time).